app/assets/stylesheets/application.scss

```diff
@@ -234,3 +234,17 @@ tr input[type=number] {
   max-height: 120px;
   min-height: 90px;
 }
+
+.chip {
+  position: absolute;
```

Comment:
Properties should be ordered background-color, color, font-size, height, line-height, padding-left, position, right, text-align, top, width